app/src/main/java/de/tum/in/tumcampusapp/component/tumui/tutionfees/TuitionFeesActivity.java

```diff
@@ -53,7 +53,15 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public void onFetch(TuitionList tuitionList) {
         String amount = tuitionList.getTuitions().get(0).getSoll();
-        amountTextView.setText(String.format("%s€", amount));
+        if (amount.contains(",")) {
+            // Make sure that the format is correct (e.g. 10,4 -> 10,40)
+            String[] parts = amount.split(",");
+            String decimals = parts[1];
+            if (decimals.length() == 1) {
+                amount += "0";
+            }
+        }
+        amountTextView.setText(String.format("%s €", amount));
```

Comment:
oh noes. This really isn't the greatest solution for this.
We actually want something along the lines of:
```java
String.format("%.2f", amount);
```
This properly handles the users locale, i.e. using a dot as decimal separator. Unfortunately needs to parse the amount string as float first...

Reply:
You’re absolutely right – I forgot about the possibility to parse as Float and then use String.format(). Fixed now.

I’m replacing the comma with a dot and then use Float.parseFloat(). The only change that this causes is that it now displays "0.00 €" instead of "0 €".

@pfent Let me know if that’s an okay solution.